### PR TITLE
ui: Add sorting to Policies

### PR DESCRIPTION
**Pull request by kaxcode**
Sort by name.

policies

### CODE REVIEW

File: ui-v2/tests/acceptance/page-navigation.feature [hunk 1]
```diff
@@ -46,11 +46,7 @@ Feature: page-navigation
...
-# These Endpoints will be datacenters due to the datacenters checkbox selectors                                               | /dc-1/acls/tokens   |
-    | policy    | policies   | /dc-1/acls/policies/ee52203d-989f-4f7a-ab5a-2bef004164ca | /v1/catalog/datacenters                                                     | /dc-1/acls/policies |
-    # | token     | tokens     | /dc-1/acls/tokens/00000000-0000-0000-0000-000000000000   | /v1/acl/token/00000000-0000-0000-0000-000000000000?dc=dc-1    | /dc-1/acls/tokens |
-    # | policy    | policies   | /dc-1/acls/policies/ee52203d-989f-4f7a-ab5a-2bef004164ca | /v1/acl/policy/ee52203d-989f-4f7a-ab5a-2bef004164ca?dc=dc-1   | /dc-1/acls/policies |
-    --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
```
Comment (kaxcode) [inline ui-v2/tests/acceptance/page-navigation.feature:49]: I decided to delete all this since we have individual navigation feature test for all acls.

Comment (johncowen) [reply ui-v2/tests/acceptance/page-navigation.feature:49]: yep good call 👍

File: ui-v2/tests/acceptance/dc/acls/policies/navigate.feature [hunk 2]
```diff
@@ -0,0 +1,15 @@
+@setupApplicationTest
```
Comment (johncowen) [inline ui-v2/tests/acceptance/dc/acls/policies/navigate.feature:1]: Didn't we call this file `navigation.feature` in the other PRs? Probably best to keep everything consistent.

File: ui-v2/tests/acceptance/dc/acls/policies/navigate.feature [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    When I visit the policies page for yaml
+    ---
+      dc: dc-1
+    ---
+    Then the url should be /dc-1/acls/policies
+    And the title should be "Policies - Consul"
+    Then I see 3 policy models
+    When I click policy on the policies
+    And I click "[data-test-back]"
+    Then the url should be /dc-1/acls/policies
```
Comment (johncowen) [inline ui-v2/tests/acceptance/dc/acls/policies/navigate.feature:15]: Didn't we add the 'click create and back' tests in the other PRs also? Actually I just checked and I don't think we did it here either? https://github.com/hashicorp/consul/pull/8359/files#r461699456 ? Maybe just leave it for now and we can circle back later. Up to you.

File: ui-v2/tests/acceptance/dc/acls/policies/navigate.feature (hunk 2 above)
Comment (kaxcode) [reply ui-v2/tests/acceptance/dc/acls/policies/navigate.feature:1]: Yes 😅 . I'll update the name.

File: ui-v2/tests/acceptance/dc/acls/policies/navigate.feature (hunk 3 above)
Comment (kaxcode) [reply ui-v2/tests/acceptance/dc/acls/policies/navigate.feature:15]: Oh yea, I just did create for intentions. Not for any fo the ACLs tabs.

Comment (kaxcode) [reply ui-v2/tests/acceptance/dc/acls/policies/navigate.feature:15]: I'll leave those out for now.

**Pull request merged**